packages/venia-ui/lib/components/CartPage/ProductListing/EditModal/productForm.js

```diff
@@ -120,49 +121,11 @@ export const GET_CONFIGURABLE_OPTIONS = gql`
         products(filter: { sku: { eq: $sku } }) {
             items {
                 id
-                sku
-                ... on ConfigurableProduct {
-                    configurable_options {
-                        attribute_code
-                        attribute_id
-                        id
-                        label
-                        values {
-                            default_label
-                            label
-                            store_label
-                            use_default_value
-                            value_index
-                            swatch_data {
-                                ... on ImageSwatchData {
-                                    thumbnail
-                                }
-                                value
-                            }
-                        }
-                    }
-                    variants {
-                        attributes {
-                            code
-                            value_index
-                        }
-                        product {
-                            id
-                            price {
-                                regularPrice {
-                                    amount {
-                                        currency
-                                        value
-                                    }
-                                }
-                            }
-                            sku
-                        }
-                    }
-                }
+                ...ProductDetailsFragment
```

Comment:
Unfortunately you cannot just re-use the fragment here and the content needed by the `RootComponent/Product` and the content needed by this component are different. Re-using the fragment will result in over-fetching of data. 

Please revert this change, or create a `productFormFragment`.